source/time_tables/departure_boards.cpp

```diff
@@ -187,8 +187,20 @@ void departure_board(PbCreator& pb_creator, const std::string& request,
                 }
             }
         }
+
         if (stop_times.empty() && (response_status.find(route->idx) == response_status.end())) {
-            response_status[route->idx] = pbnavitia::ResponseStatus::no_departure_this_day;
+            if (rt_level == navitia::type::RTLevel::Base) {
+                response_status[route->idx] = pbnavitia::ResponseStatus::no_departure_this_day;
+            } else {
+                auto tmp_stop_times = routing::get_stop_times(routing::StopEvent::pick_up, routepoint_jpps, handler.date_time,
```

Comment:
long line, and it could be great to add a comment explaining why we do this call.